### PR TITLE
Jen/facility name description

### LINKED ISSUE #188: Facility name and details section - Multi-facility polish
**What needs to be done? Why does it need to be done?**
Add facility name, details as shown in mocks

**Additional context**
Subtask Issue #167

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Updates the `FacilityInputForm` Name and Description inputs to match the mockups. 
-Creates a generic `InputNameWithIcon` component as the editable scenario/facility name header
-Adds description attribute to the Facility model
-Adds `InputDescription` component and add it to `FacilityInputForm`

Note: I saw that @daschi was working on the description section of the `ScenarioSidebar` so I pulled in her commits and addressed the comment about `onEnterPress` event. This PR is based off of that branch.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #188 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jessex** [comment]
Just a heads up that we've got Justin's recent changes pulled into here as part of the diff, so we may want to rebase/merge them out of here to simplify the PR!

File: src/page-multi-facility/ScenarioSidebar.tsx [hunk 1]
```diff
@@ -3,53 +3,12 @@ import React, { useState } from "react";
 import styled from "styled-components";
 
 import Colors from "../design-system/Colors";
-import iconEditSrc from "../design-system/icons/ic_edit.svg";
```
Comment (jessex) [inline src/page-multi-facility/ScenarioSidebar.tsx:6]: Looks like we have a couple of outdated imports the linter caught that we can remove now.

**Comment by jovergaag** [comment]
@jessex I had noticed the sluggish inputs locally too. I wanted to have it deployed to a review app to see if it is only in the local environment. I'll keep digging into this.

**Comment by jessex** [comment]
Thanks for investigating, @jovergaag! I have to imagine it's somehow related to updates to any field within the facility getting pulled into an auto-updated of the epidemic model, but that's purely speculative on my part.

**Comment by macfarlandian** [comment]
Every keystroke in a field that feeds into an `EpidemicModelContext` causes a recalculation of the curves, which is not great! We should probably debounce those input changes but we just never got around to it.

**Comment by jovergaag** [comment]
@macfarlandian That explains why name/description inputs on this page are so sluggish. I'll fix that today. Thanks!

**Comment by jovergaag** [comment]
@jessex @macfarlandian This is ready for another look.

**Pull request merged**